geode-core/src/main/java/org/apache/geode/cache/client/internal/OpExecutorImpl.java

```diff
@@ -556,8 +556,8 @@ protected void handleException(Throwable e, Connection conn, int retryCount, boo
       title = null;
       exToThrow = new ServerOperationException(e);
     } else if (e instanceof SerializationException) {
-      title = null; // no message
-      exToThrow = new ServerOperationException(e);
+      title = "Unexpected SerializationException";
+      // exToThrow = new ServerOperationException(e);
```

Comment:
you should remove this commented-out code

Reply:
We have a new fix.